recipes-devtools/visionworks/libvisionworks_1.6.0.501.bb

```diff
@@ -27,15 +27,15 @@ do_install() {
     install -d ${D}${prefix} ${D}${libdir} ${D}${datadir} ${D}${libdir}/pkgconfig
     cp -R --preserve=mode,timestamps ${B}/usr/include ${D}${prefix}/
     cp -R --preserve=mode,timestamps ${B}/usr/lib/pkgconfig/* ${D}${libdir}/pkgconfig
-    cp --preserve=mode,timestamps ${B}/usr/lib/libvisionworks.so ${D}${libdir}/
+    cp --preserve=mode,timestamps ${B}/usr/lib/libvisionworks.so* ${D}${libdir}/
```

Comment:
This should also add `,links` to the `--preserve` and add `--no-dereference`, so the symlinks are preserved as-is.